dependencies.properties

```diff
@@ -39,8 +39,24 @@ maven.com_google_auth_google_auth_library_credentials=com.google.auth:google-aut
 maven.io_opencensus_opencensus_api=io.opencensus:opencensus-api:0.24.0
 maven.io_opencensus_opencensus_contrib_grpc_metrics=io.opencensus:opencensus-contrib-grpc-metrics:0.24.0
 maven.io_opencensus_opencensus_contrib_http_util=io.opencensus:opencensus-contrib-http-util:0.24.0
+maven.io_netty_netty_tcnative_boringssl_static=io.netty:netty-tcnative-boringssl-static:2.0.36.Final
```

Comment:
Can anyone explain why this upgrade requires me to add all these transitive dependencies from gRPC? Something about bazel builds I'm missing? @ejona86

Reply:
Seems you aren't using mvn_install: https://github.com/googleapis/gax-java/blob/master/WORKSPACE

https://github.com/grpc/grpc-java/releases/tag/v1.33.0

> bazel: Remove Maven repositories from repositories.bzl, in favor of maven_install. v1.27.0 introduced support for maven_install and encouraged users to migrate. See examples/WORKSPACE for an example. maven_install dramatically reduces the boilerplate for maven dependencies and properly handles transitive dependencies and version selection. gRPC is not yet using the `@maven` workspace, so it is still possible to use other dependency tools.

https://github.com/grpc/grpc-java/releases/tag/v1.27.0

> Bazel users should migrate to maven_install. Please open an issue to report problems migrating to maven_install.

> bazel: Support maven_install (#6553). See examples/WORKSPACE for typical usage. grpc-java still supports the jvm_maven_import_external()-based loading only to ease migration. grpc-java will drop usages of jvm_maven_import_external() in the future